netlify/functions/submit-finding.ts

```diff
@@ -88,10 +88,14 @@ exports.handler = async (event) => {
       }),
     };
   }
+  const isMitigationReport : boolean = mitigationOf ? true : false;
+  let mitigationLabels = ['mitigation-confirmed', `MR-${mitigationOf}`];
+  let mitigationTitle = `Mitigation Confirmed for ${mitigationOf}`;
```

Comment:
This is a minor thing, but it would be a good idea to make these values "falsey" if `isMitigationReport` is false.

This would make both values `false` if not a mitigation report
```javascript
  const mitigationLabels = isMitigationReport && ['mitigation-confirmed', `MR-${mitigationOf}`];
  const mitigationTitle = isMitigationReport && `Mitigation Confirmed for ${mitigationOf}`;
```

An alternative which would make both values `undefined` if not a mitigation report
```javascript
  const mitigationLabels = isMitigationReport ? ['mitigation-confirmed', `MR-${mitigationOf}`] : undefined;
  const mitigationTitle = isMitigationReport ? `Mitigation Confirmed for ${mitigationOf}` : undefined;
```